README.md

```diff
@@ -90,14 +91,34 @@ region using an environment variable.
 export AWS_REGION=eu-west-1
 ```
 
+## Getting Started with aws
+
+To stand up an instance with AWS, use the `ec2_instance` type:
```

Comment:
this example actually creates 2 instances. It's also missing the region property, and requires a security group. The image_ids are also somewhat specific to the example it's taken from. The tagging is also very Puppet Labs specific. The deleted example is much better in my view.